app-schema-links.md

```diff
@@ -0,0 +1,3 @@
+# SomeUniqueBananas
+
+- app schema `sub://` 
```

Comment:
Isn't `sub` likely to be reserved by some other app? What about `subananas://`?

Reply:
Great catch, I'll fix